### PR TITLE
OTel in Focus Jan 2023

### PR DESCRIPTION
**Pull request by cartermp**
https://docs.google.com/document/d/19kjBtzaLnHT-R3_RJL40ooZMBIfg-Sb7UuTWJ4qlQvE/edit#

Preview:

- https://deploy-preview-2237--opentelemetry.netlify.app/blog/
- https://deploy-preview-2237--opentelemetry.netlify.app/blog/2023/otel-in-focus-01

### CODE REVIEW

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+notes](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit),
+for more information.
+
+The [End User Working
+Group](https://opentelemetry.io/blog/2023/otel-end-user-discussions-january-2023/)
+has published a summary of their discussions for the month. If you’re a user of
+OpenTelemetry, these meetings are a great place to get connected with your peers
+and discuss how you’re using the project.
+
+## News andUpcoming Events
```
Comment (austinlparker) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:79]:
```suggestion
## News and Upcoming Events
```

**Comment by austinlparker** [comment]
Thanks! We won't have to do this next time since y'all can approve now...

**Review by austinlparker — APPROVED** [review]

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+title: OpenTelemetry in Focus - January 2023
+linkTitle: OpenTelemetry in Focus - January 2023
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:3]:
```suggestion
linkTitle: OTel in Focus - 2023/01
```

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+as part of KubeCon EU 2023. If you’re planning to be in Amsterdam for KubeCon,
+be sure to come a day early and meet up with OpenTelemetry contributors and
+maintainers!
+
+
+
+
+
+
+
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:92]:
```suggestion
```

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
+be an overview of important releases, roadmap updates, and community news. This
+series will be focusing on our core components, such as the specification, data
+format, tools, and most popular API/SDKs.
+
+Are you a maintainer with something you’d like featured here? Please get in
+touch with me via email, or on the CNCF Slack!
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:15]: In addition to the suggestion given below, consider adding the link to the CNCF Slack.

```suggestion
Are you a maintainer with something you’d like featured here? Get in
touch with me via email, or on the CNCF Slack!
```

How are folks meant to get your email address?

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+- [Demo
+v1.3.0](https://github.com/open-telemetry/opentelemetry-demo/releases/tag/1.3.0)
+has been released with support for metric exemplars, enhanced resource
+detection, and updates to OpenTelemetry API and SDKs.
+
+## Project Updates
+
+Over the past few months, through community discussions both in-person and
+online, a [new public
+roadmap](https://github.com/open-telemetry/community/blob/main/roadmap.md) has
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:62]: I agree with @svrnm, that we should link to the Roadmap on the website. To this end, I've created the following simple PR:

- #2239

The above is propose as a quick addition until we can get #2176 resolved.

WDYT?
```suggestion
online, a new [public roadmap](/community/roadmap/) has
```

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
+be an overview of important releases, roadmap updates, and community news. This
+series will be focusing on our core components, such as the specification, data
+format, tools, and most popular API/SDKs.
+
+Are you a maintainer with something you’d like featured here? Get in
+touch with me [via email](austin@lightstep.com), or on the CNCF Slack!
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:15]:
```suggestion
touch with me [via email](mailto:austin@lightstep.com), or on the CNCF Slack!
```

File: content/en/blog/2023/opentelemetry-in-focus-january-2023.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (chalin) [inline content/en/blog/2023/opentelemetry-in-focus-january-2023.md:1]: Consider renaming the file to: `content/en/blog/2023/otel-in-focus-2023-01.md` (we could even drop the `2023-` and just use the name `2023/otel-in-focus-01.md`)

File: content/en/blog/2023/otel-in-focus-01.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
+---
+title: OpenTelemetry in Focus - January 2023
+linkTitle: OTel in Focus - 2023/01
+date: 2023-01-31
+author: >-
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-01.md:9]:
```suggestion
Welcome to the first edition of _OpenTelemetry In Focus_! This blog is intended to
```

File: content/en/blog/2023/otel-in-focus-01.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+connectors in the Collector Builder.
+- [Demo
+v1.3.0](https://github.com/open-telemetry/opentelemetry-demo/releases/tag/1.3.0)
+has been released with support for metric exemplars, enhanced resource
+detection, and updates to OpenTelemetry API and SDKs.
+
+## Project Updates
+
+Over the past few months, through community discussions both in-person and
+online, a new [public roadmap](/community/roadmap/) has
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-01.md:61]:
```suggestion
online, a new public [roadmap](/community/roadmap/) has
```

File: content/en/blog/2023/otel-in-focus-01.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+  [Austin Parker](https://github.com/austinlparker)
+---
+
+Welcome to the first edition of OpenTelemetry In Focus! This blog is intended to
+be an overview of important releases, roadmap updates, and community news. This
+series will be focusing on our core components, such as the specification, data
+format, tools, and most popular API/SDKs.
+
+Are you a maintainer with something you’d like featured here? Get in
+touch with me [via email](mailto:austin@lightstep.com), or on the CNCF Slack!
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-01.md:15]:
```suggestion
touch with me [via email](mailto:austin@lightstep.com), or on the
[CNCF Slack, #otel-comms](https://cloud-native.slack.com/archives/C02UN96HZH6) channel.
```

https://cloud-native.slack.com/archives/C02UN96HZH6

**Review by chalin — APPROVED** [review]
LGTM

See inline for a few final teaks (e.g., added link to Slack channel).

File: content/en/blog/2023/otel-in-focus-01.md (hunk 10 above)
Comment (chalin) [reply content/en/blog/2023/otel-in-focus-01.md]: ☝️ @austinlparker - is it ok to refer folks to the `#otel-comm` channel, or were you expecting a DM via Slack?

**Pull request merged**